lib/worker.coffee

```diff
@@ -20,7 +20,7 @@ module.exports = (text, attributes, options) ->
 
   Opal.ENV['$[]=']('PWD', path.dirname(options.opalPwd))
```

Comment:
This environment variable is no longer used in Asciidoctor.js we are using the default implementation provided by Opal: https://github.com/opal/opal/blob/7716df4f0afe981ecbec9e0c9a18750d3de10aed/stdlib/nodejs/dir.rb#L12-L14

You may run into troubles because I doubt that `process.cwd()` is equals to `window.location.href` ? :thinking: